src/train/pipeline.py

```diff
@@ -13,7 +13,7 @@
 from extractor import DefaultExtractor
 from isolator import MinIdleIsolator
 
-from train_types import PowerSourceMap, FeatureGroups, ModelOutputType
+from train_types import PowerSourceMap, FeatureGroups, ModelOutputType, ACCELERATE_FEATURES
```

Comment:
ACCELERATE_FEATURES should be referred under FeatureGroup (same as other metric group).

Reply:
Thanks! I will fix it.

Reply:
Fixed all! Thanks!